### DISCUSSION TITLE
Screen goes blank after 30 seconds

### DISCUSSION THREAD

**Discussion by Kronzky** [devemux86/cruiser#605] [Q&A] [answered]
No matter whether I'm following a route or just cruising around, my screen will go blank after 30 seconds.

I'm using the latest Beta (4.1.0-b2) on a Pixel 4a with Android 14, the Screen saver option under Navigation is turned off, and battery usage is set to unrestricted in the Android settings

**Comment by DolomitesTom** [comment]
You have to deactivate the screen saving in settings -- navigation.

**Comment by devemux86** [comment, marked as answer]
@Kronzky 

- Is Cruiser screen saver activated in "Settings | Navigation | Screen saver"?
(you mention it is off)

- You can keep the screen on in "Settings | Application |Screen | Keep screen on".

**Reply by Kronzky** [reply]
> * You can keep the screen on in "Settings | Application |Screen | Keep screen on".

That was it! (I haven't tried it on the road yet, but so far it's staying on.)

But you **really** need some documentation for this software...
I know, most developers hate to write documentation, but I've been in the business for over 30 years, and during that time I've learned that the time spent writing proper documentation always pays off; as you then don't have to answer the same questions over and over again. Especially if the software is as complex as yours. 
Plus, most users will just give up right away (and use something else) if features aren't obvious or well documented.

**Reply by 0709wiwiwi** [reply]
- Contributions welcome  😊
https://github.com/devemux86/cruiser/wiki
https://github.com/devemux86/cruiser/discussions/490

Find some usefull how to examples here.
https://github.com/devemux86/cruiser/wiki/How-to%3F
